<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QDialog" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SignUp</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="lblUserName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>44</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>User Name :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtbSureName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Last Name
</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none ;
border-radius:4px;</string>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblFirstName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtbFirstName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>First Name
</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none ;
border-radius:4px;</string>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblSureName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>244</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Sure Name :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtbUserName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>*user name with at least 8 and at most 12 characters
</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none ;
border-radius:4px;</string>
   </property>
   <property name="maxLength">
    <number>12</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblPassword">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>83</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Your Password :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtbPassword">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>*user name with at least 8 and at most 12 characters
</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none ;
border-radius:4px;</string>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblRePass">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>121</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Renter Your Password :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtbRePass">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>*user name with at least 8 and at most 12 characters
</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none ;
border-radius:4px;</string>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>424</x>
     <y>340</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none ;
border-radius:4px;</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>340</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none ;
border-radius:4px;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>322</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtbUserName</tabstop>
  <tabstop>txtbPassword</tabstop>
  <tabstop>txtbRePass</tabstop>
  <tabstop>txtbFirstName</tabstop>
  <tabstop>txtbSureName</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
